packages/website/docs/Api/Plot.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 8
+title: Plot
+---
+
+Plot objects can be created to make plots of different kinds. If you wish to plot
+multiple distributions simultaneously, you can use `Plot.dist`.
+
+**Example**
+
+### dist
+
+```
+Plot.dist({show: list({name: string, value: distribution|number})})
```

Comment:
What do you think future args will be, besides ``show``?

I imagine "show" is a bad name. I'd probably use "dists", "items", or similar. Imagining that other inputs would be things like "title" and "params" or something. The title would be "shown".

Reply:
Also, I'm curious about using "title" instead of "name". I'd be curious to see how its done in other similar libs. 

I particularly care about getting the API right, because that would be a pain to change later.

Reply:
Another idea could be to name this "Plot.dists"; if we think we'll want a version just for a single dist later on. (This seems pretty minor to me though)